Client/Frontend/Browser/BrowserViewController/BrowserViewController+ToolbarDelegate.swift

```diff
@@ -208,10 +208,13 @@ extension BrowserViewController: TopToolbarDelegate {
 
   func processAddressBar(text: String, visitType: VisitType, isBraveSearchPromotion: Bool = false) {
     if let fixupURL = URIFixup.getURL(text), !isBraveSearchPromotion {
-      // The user entered a URL, so use it.
-      finishEditingAndSubmit(fixupURL, visitType: visitType)
-
-      return
+      // Do not allow users to enter URLs with the following schemes.
+      // Instead, submit them to the search engine like Chrome-iOS does.
+      if !["file"].contains(fixupURL.scheme) {
```

Comment:
im just curious, why isn't this just checking if fixupURL.scheme ~= "file"? Is there some corner case im not thinking of?  Are there other "file-y" schemes this is catching?

Reply:
No reason. I could check if `fixupURL.scheme != "file"` but I chose to make it an array so in the future if we wanted to add other schemes here to block, we could. I was just following the convention we already do in other parts of the code for `["data", "file", "http", "https", "blob"]`